cmd/server/README.md

```diff
@@ -8,22 +8,44 @@ The Credential Server serves the User Service, which is defined in an [OpenAPI 3
 
 Make sure you have [Bazel](https://bazel.build/) installed. For more detailed instructions on some of the tools we use and how we use them, check out the [Silicon Ally Developer Handbook](https://siliconally.getoutline.com/s/d984f195-3e5e-410f-bce8-63676496661f).
 
-Once that's done, run the following to generate the key pair used for JWT signing + validation. It'll create `test_server.{pub,key}` files in the root of the project.
+Once that's done, you'll either need access to the sops-encrypted `cmd/server/configs/secrets/local.enc.json` file or otherwise replace it with your own, which should contain the contents:
```

Comment:
perhaps link to something describing what this would entail/how to request it.

Reply:
Good call, added some color here.